src/Microsoft.AspNet.Razor.Test.Sources/CaseSensitiveTagHelperDescriptorComparer.cs

```diff
@@ -6,9 +6,9 @@
 using Microsoft.AspNet.Razor.TagHelpers;
 using Microsoft.Internal.Web.Utils;
 
-namespace Microsoft.AspNet.Razor.Runtime.TagHelpers
+namespace Microsoft.AspNet.Razor.Test.Internal
```

Comment:
Shouldn't have `.Test` in the namespace except for test infrastructure classes. So `.Internal` seems redundant. Why not just `Microsoft.AspNet.Razor.Test`?